typeid/typeid.py

```diff
@@ -80,15 +80,14 @@ def from_uuid(suffix: uuid6.UUID, prefix: Optional[str] = None) -> TypeID:
 
 
 def get_prefix_and_suffix(string: str) -> tuple:
-    parts = string.split("_")
-    prefix = None
+    parts = string.rsplit("_", 1)
```

Comment:
I am suggesting this because of the next comment.

```suggestion
    parts = string.split("_")
```